lib/honeycomb-rails/railtie.rb

```diff
@@ -18,6 +18,10 @@ class Railtie < ::Rails::Railtie
     # the app's config/initializers has taken effect.
     config.after_initialize do
       writekey = HoneycombRails.config.writekey
+      if writekey.blank?
+        HoneycombRails.config.logger.warn("No write key defined! No events will be sent to Honeycomb.")
```

Comment:
Maybe the warning could point how to fix it? e.g.

`.warn("No write key defined! (conf.writekey in config/initializers/honeycomb.rb) No events...")`

(Little tricky since they could choose to put the initialization code in other places besides `config/initializers/honeycomb.rb`, and they could name the config variable `c` or `config` rather than `conf`, but at least that message would be accurate for what we suggest in the [docs](http://www.rubydoc.info/gems/honeycomb-rails/HoneycombRails#configure-class_method), readme and guide.)